.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-Issue #, if available:
+**Issue # (if avilable):**
```

Comment:
avil able. Also, folks will need to add the hash themselves, and Issue #: #123 stutters. Perhaps we can just do 
```
**Issue #**
```